File: Programming-languages-and-laboratory/Java/Exercises/10_compito_java_2016_07_04/Privato.java
import java.util.*;

public class Privato extends Cliente {

    private String nomeCognome;

    public Privato(int c, String nome, float p, Filiale att, List<Filiale> pass) {
        super("privato", c, p, att, pass);
        nomeCognome = nome;
    }

    public String toString() {
        return tipo + "\t" + nomeCognome + "\t" + super.toString();
    }

}

File: Programming-languages-and-laboratory/Python/Exercises/10_compito_java_20160704/Privato.py
from Cliente import Cliente


class Privato(Cliente):
    def __init__(self, tipo, codice, nome_cognome, codice_filiale_corrente):
        super().__init__(tipo, codice, codice_filiale_corrente)
        self._nome_cognome = nome_cognome
        
    def get_nome(self):
        return self._nome_cognome

    def __str__(self):
        return self._tipo + "\t" + self._nome_cognome + "\t" + str(self._codice)
